assemble/bin/accumulo-cluster

```diff
@@ -244,7 +244,7 @@ function kill_all() {
   done
 
   echo "Cleaning all server entries in ZooKeeper"
-  ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -master -tservers -tracers --site-file "${conf}/accumulo-site.xml"
+  ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -master -tservers -tracers
```

Comment:
Does the `ZooZap` command no longer need access to the site file? I would imagine it is one of those utilities that needs the `instance.secret`. I don't think a client configuration file will be an adequate substitute.

Reply:
`ZooZap` doesn't use `ClientOpts` anymore which required the site file path to be passed in. It still uses the site file as `SiteConfiguration.getInstance()` is called.

Reply:
Okay. That code will grab the site file from the class path, which is good.